games-kids/tuxmath/tuxmath-2.0.3-r3.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2025 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
 inherit desktop optfeature
```

Comment:
Please add https://github.com/tux4kids/tuxmath to HOMEPAGE & remote-id. Send the patch upstream too.